std/digest/package.d

```diff
@@ -1212,3 +1212,112 @@ if (isInputRange!R1 && isInputRange!R2 && !isInfinite!R1 && !isInfinite!R2 &&
         assert(!secureEqual(hex1, hex2));
     }
 }
+
+/**
+ * Used to convert hex string to byte array
+ * Params:
+ *  hex = hex string
+ *  num = count of bytes
+ * Returns: byte array
+ * Example:
+ * ---
+ * ubyte[2] sba = "0xAA".fromHexString!2; // returns static array, no allocating
+ * ubyte[] dby  = "0xBA".fromHexString;   // returns dynamic array, gc allocating
+ * ---
+ */
+auto fromHexString(const string hex) pure nothrow @safe
```

Comment:
I wouldn't use `auto` here since the return type is `ubyte[]`, but really it should be a range (in which case, yes, `auto`).

Reply:
```suggestion
ubyte[] fromHexString(const string hex) pure nothrow @safe
```